v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/transforms/KafkaTransform.java

```diff
@@ -126,7 +126,7 @@ public static KafkaIO.Read<byte[], byte[]> readBytesFromKafka(
                 ByteArrayDeserializer.class, NullableCoder.of(ByteArrayCoder.of()))
             .withValueDeserializerAndCoder(ByteArrayDeserializer.class, ByteArrayCoder.of())
             .withConsumerConfigUpdates(config);
-    if (sslConfig != null) {
+    if (!sslConfig) {
```

Comment:
We shouldn't need the `sslConfig` parameter at all. I suggest just setting `.withConsumerFactoryFn` unconditionally. This Fn only post-processed the config and replaces remote files/secrets with local files. If there is nothing to replace in the config, it's going to be a no-op. If there is something to replace, it'll replace that.

Reply:
Done